pkg/porter/archive_test.go

```diff
@@ -51,3 +52,18 @@ func TestArchive_Validate(t *testing.T) {
 		})
 	}
 }
+
+func TestArchive_ArchiveDirectory(t *testing.T) {
+	p := NewTestPorter(t)
+	defer p.Close()
+	ex := exporter{
+		fs: p.FileSystem,
+	}
+
+	dir, err := ex.createArchiveFolder("examples/test-bundle-0.2.0")
+	require.NoError(t, err)
+
+	info, err := ex.fs.Stat(dir)
```

Comment:
Can you add one more assert that checks the name created? That will help check for replacing the path seperator with a dash.